app/src/chainlit.py

```diff
@@ -25,14 +30,78 @@ async def start() -> None:
         ).send()
         return
 
-    cl.user_session.set("chat_engine", engine)
+    settings = await _init_chat_settings(engine)
     await cl.Message(
         author="backend",
-        metadata={"engine": engine_id},
-        content=f"Chat engine started: {engine.name}",
+        metadata={"engine": engine_id, "settings": str(settings)},
+        content=f"{engine.name} started with settings:\n{pprint.pformat(settings, indent=3)}",
```

Comment:
Printing the settings for internal team use. We can remove this when piloting with actual users.

Reply:
Do we need this if it's already in the logger?

Reply:
It's intended for non-engineers as a reminder of the current setting values, since these settings are newly added. Once the team understands this, we can reduce the verbosity of this.